docs/architecture/adr-020-deterministic-square-construction.md

```diff
@@ -80,6 +80,6 @@ The new algorithm will no longer need to check that the blobs are ordered by nam
 
 ## References
 
-- [ADR013](./adr-013-non-interactive-default-rules-for-zero-padding.md)
+- [ADR-013](./adr-013-non-interactive-default-rules-for-reduced-padding-2.md)
```

Comment:
```suggestion
- [ADR013](./adr-013-non-interactive-default-rules-for-reduced-padding-2.md)
```
What was the reasoning for changing the format here. It seem elsewhere we don't include the `-`

Reply:
I was attempting to be consistent with how ADRs are referenced and always use the hyphen. Will revert the name change and link changes.